src/test/dialogs.test.js

```diff
@@ -0,0 +1,209 @@
+var expect = require("chai").expect;
+var React = require("react/addons");
+var TestUtils = React.addons.TestUtils;
+
+var DialogActions = require("../js/actions/DialogActions");
+var DialogEvents = require("../js/events/DialogEvents");
+var DialogStore = require("../js/stores/DialogStore");
+
+var expectAsync = require("./helpers/expectAsync");
+
+describe("Dialog store", function () {
+
+  describe("alert", function () {
+
+    it("show sends correct message", function (done) {
+      DialogActions.alert("test alert");
```

Comment:
This is meant to be here, because the DialogEvents are triggered in the next event cycle by design. So I expect, that this order have to work.

See
https://github.com/mesosphere/marathon-ui/blob/master/src/js/AppDispatcher.js#L3